roachpb/internal.proto

```diff
@@ -125,6 +125,8 @@ message RaftSnapshotData {
   message KeyValue {
     optional bytes key = 1;
     optional bytes value = 2;
+    optional int64 wall_time = 3 [(gogoproto.nullable) = false];
```

Comment:
Probably. I had an import issue with the protos and meant to revisit this. I had used separate fields purely for expediency during development.